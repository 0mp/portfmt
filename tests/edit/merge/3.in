PORTNAME=	sndio

USERS=		_sndio
