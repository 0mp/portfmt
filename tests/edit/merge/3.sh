printf "PORTNAME=tokei\nUSES=gmake\nCARGO_CRATES=\nCARGO_USE_GITHUB=\n" | ${PORTEDIT} merge 3.in | \
	diff -L 3.expected -L 3.actual -u 3.expected -
