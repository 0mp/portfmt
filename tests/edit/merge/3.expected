PORTNAME=	tokei

USES=		gmake
CARGO_CRATES=
CARGO_USE_GITHUB=
USERS=		_sndio
